divisible.rb

```diff
@@ -0,0 +1,13 @@
+def divisible
+    i=1
+    res=[]
+    while i<=100 do
+        if(i%2==0 || i%3==0 || i%5==0)
+            res.push(i)
+        end
+        i=i+1
+    end
+    return res
+end
+
+divisible()
```

Comment:
You're missing the part where you print the result, but I'll accept it anyway this time.

```suggestion
puts divisible()
```